### DISCUSSION TITLE
Pipeline per-branch in a given repository

### DISCUSSION THREAD

**Discussion by jndietz** [concourse/concourse#6076] [Help & Support] [answered]
What would be the easiest way to automatically create pipelines for each branch in a single Git repository?  I have an idea of how to do it, but how could I organize it on the UI without creating a giant list of pipelines?

**Comment by vito** [comment, marked as answer]
Check out the [v10 roadmap](https://github.com/concourse/concourse#the-road-to-concourse-v10) if you haven't seen it yet - it's geared for exactly this purpose.

Once pipeline instances is done (#5896 + #5921) that should resolve the UI problem of organizing a bunch of pipelines.

There are still some pieces that need to be implemented though - namely, dynamic `across` steps.
